css/style.css

```diff
@@ -16,4 +15,199 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* --------------------------------------------------------------------------------------------------------- */
+
+/* 1.0 HEADER PART */
+
+/* 1.1 making space between the contents in header  */
+header {
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
```

Comment:
You can add `align-items: center;` here to vertically align both the logo and the nav.